packages/web3/src/utils/utils.ts

```diff
@@ -63,7 +63,7 @@ export function xorByte(intValue: number): number {
 }
 
 export function isHexString(input: string): boolean {
-  return input.length % 2 === 0 && /[0-9a-f]*$/.test(input)
+  return input.length % 2 === 0 && /^[0-9a-fA-F]*$/.test(input)
```

Comment:
Full node does not support uppercase hexstinrg. If we want to change it, we should change both. Maybe we should support prefix too.

Reply:
> Full node does not support uppercase hexstinrg.

Do you mean the [Hex](https://github.com/alephium/alephium/blob/55f49f3939cf22b4173b0a1bc0dd92a1e6cf211b/util/src/main/scala/org/alephium/util/Hex.scala#L25)? Just tested it, it supports uppercase hex string.

Reply:
Okay, I thought it does not. Then let's merge this first.